### PR TITLE
Add Mouse Fix 1.0.1

### PR DESCRIPTION
**Pull request by ran-dall**
🚧 Readdresses #85868

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew install --cask {{cask_file}}` worked successfully.
- [x] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by miccal** [comment]
@ran-dall any chance you can add a `zap`?

File: Casks/mouse-fix.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+cask "mouse-fix" do
+  version "1.0.1"
+  sha256 "923469ae24f796115eebab2baf71d05e0c6b0fef0a5a8deb2afe800c83995044"
+
+  url "https://github.com/noah-nuebling/mac-mouse-fix/releases/download/#{version}/MacMouseFixApp.zip",
```
Comment (vitorgalvao) [inline Casks/mouse-fix.rb:5]: It’s a GitHub releases URL, so `livecheck` works without explicitly adding the block. This isn’t true alway, but it is most of the time (including in this case).

**Review by vitorgalvao — APPROVED** [review]

**Comment by noah-nuebling** [comment]
This is really cool I love brew :)
Thanks again guys!

File: Casks/mouse-fix.rb (hunk 1 above)
Comment (ran-dall) [reply Casks/mouse-fix.rb:5]: Noted

**Pull request merged**